pom.xml

```diff
@@ -66,7 +66,7 @@ limitations under the License.
     <!-- hbase dependency versions -->
     <hbase1.version>1.7.1</hbase1.version>
     <hbase1-metrics.version>3.1.2</hbase1-metrics.version>
-    <hbase2.version>2.2.3</hbase2.version>
+    <hbase2.version>2.2.6</hbase2.version>
```

Comment:
is this the right version? PR title says 2.3.6?

Reply:
Ah yeah I made a mistake, should be 2.3.6